app/controllers/timer_sessions_controller.rb

```diff
@@ -5,15 +5,8 @@ def index
     @timer_sessions_in_range = TimerSession.includes(:issues, :time_entries, :timer_session_time_entries)
                                            .finished
                                            .created_by(@current_user)
-    @non_matching_timer_session_ids = TimeDiscrepancyLoader.uneven_timer_sessions(@timer_sessions_in_range).pluck(:id)
-    @timer_sessions = apply_filter(@timer_sessions_in_range, :timer_start)
-    time_entries = time_entries_in_range(@timer_sessions)
-
-    @timer_sessions = (@timer_sessions.order(timer_start: :desc) + time_entries)
-                      .map { |time_entity| TimeEntityDecorator.new(time_entity) }
-                      .sort_by(&:start_time)
-                      .reverse
-                      .group_by(&:entry_date)
+    @non_matching_timer_session_ids = TimeDiscrepancyLoader.uneven_timer_sessions(@timer_sessions_in_range).map(&:id)
+    timer_sessions
```

Comment:
Please express what you're doing with the name of the method. `set_timer_sessions` is better.

Reply:
I haven't renamed any methods

Reply:
Shouldn't getters and setters avoid using set_ in ruby?

Reply:
You're mixing up two things:

* Setting instance values from outside the class:
  We rather use setters in Ruby like this
  ```rb
  def timer_sessions=(value)
    @whatever = value
  end
  ```
  than like this
  ```rb
  def set_timer_sessions(value)
    @whatever = value
  end
  ```
* Setting instance values from inside the class:
  Calling `timer_sessions` just like this doesn't expose any instance altering behaviour to the reader,
  but something like `set_timer_sessions`, `configure_timer_sessions` or `setup_timer_sessions` would.